flybywiresim/a32nx
# fix: add package-lock json and revert broken dependency

Fixes #6897

## Summary of Changes

Added package-lock.json to pin broken dependency used by the MCDU Server.

Backup plan if https://github.com/tomas/wmic/issues/19 isn't published quick enough.
## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions

1. Make sure the MCDU Server is starting properly again
2. Load up the plane and ensure all systems and screens are on and functioning.
3. Verify Remote MCDU works as before.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
